### PR TITLE
Feature/19 update entity model

### PR DESCRIPTION
**Pull request by RamonG92**

### CODE REVIEW

File: ormconfig.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "host": "localhost",
     "port": 3306,
     "username": "root",
-    "password": "",
+    "password": "root",
```
Comment (misharigot) [inline ormconfig.json]: Maybe we can find a better solution for this, maybe an ormconfig.json.dist and then ignore this one? Or inject variables via some other config file, e.g. .env if possible.

Comment (misharigot) [reply ormconfig.json]: Found something, https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md#using-environment-variables

**Comment by misharigot** [comment]
@RamonG92 will setup a migrations workflow and documentation around it.

File: src/entities/attachment.entity.ts [hunk 2]
```diff
@@ -4,27 +4,39 @@ import { Log } from './log.entity';
...
-    file_md5: string;
-}
\ No newline at end of file
+    @Column({
+        name: 'file_md5',
+        type: 'char',
+        length: 16,
+        nullable: true
+    })
+    fileMd5: string;
```
Comment (misharigot) [inline src/entities/attachment.entity.ts]: I think for camel casing two letter abbreviations can be with two capital letters, i.e. fileMD5, just like fileIO.

Comment (RamonG92) [reply src/entities/attachment.entity.ts]: Good catch, changed it to fileMD5.

**Pull request merged**